### ISSUE TITLE
Flash player should have white background

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#4090]
**Describe the bug**
Currently, when the player is a flash player (as in IE11, Windows 7, and others), the background is black. Because of this, the player can either not see the controls, or only slightly see them, so they fail the contrast ratio necessary for accessibility.

**To Reproduce**

<img width="685" alt="Screenshot 2019-09-19 at 15 39 57" src="https://user-images.githubusercontent.com/54575620/66132483-0aa36100-e5ed-11e9-9464-99618d502c97.png">

1. Go to Windows 7 IE11
2. Click on play
3. See black background and controls

**Expected behavior**
Flash player should have a white background so that the controls have a high enough contrast.

**Testing notes**
Test the controls on the player have a high enough contrast with the background to pass accessibility test.
- [x] This bug fix is expected to need manual testing.

@ryanmccombe  You did some work on morph modules for this?

**Comment by pjlee11** [comment]
The issue should have been fixed by https://github.com/bbc/morph-modules/pull/32409, which is currently deployed to the TEST morph env. 

Therefore, can we test the flash player no longer have a black background on the TEST env - www.test.bbc.com/korean/bbc_korean_radio/liveradio before we promote the changes to LIVE

**Comment by LilyL0u** [comment]
Using Browserstack, Windows 7 IE11, flash player still has black background:

![Image 04-10-2019 at 12 03](https://user-images.githubusercontent.com/54575620/66203037-2ff0a780-e69f-11e9-9700-12b7b497aded.jpg)

I found a real computer to test this on, and it is also still a black background. @pjlee11

**Comment by ryanmccombe** [comment]
This should now be ready to test

Have deployed it just to TEST initially - eg https://www.test.bbc.com/korean/bbc_korean_radio/liveradio - if all seems good there, we can promote to live

From a quick check, it does seem like everything still works.  It's also loading in the TEST version of SMP.  This should include Jim's fix for the black background, but I can't verify that on my computer

cc @LilyL0u

**Comment by LilyL0u** [comment]
It has a white background on Browserstack now :)

![UNADJUSTEDNONRAW_thumb_4f](https://user-images.githubusercontent.com/54575620/66779007-22b89180-eec5-11e9-9086-9d0ca542483f.jpg)

I can move this issue on as passed, or I should wait to test it on a real computer?

**Comment by ryanmccombe** [comment]
Nice - that looks better than I thought it would

**Comment by gavinspence** [comment]
![IE11](https://user-images.githubusercontent.com/52488258/67024630-a3150780-f0fc-11e9-9d3d-d40123b2ca6b.PNG)

Just tested on one of the Windows machines, and it looks good to me. Even shows the Flash warning as expected.

**Comment by LilyL0u** [comment]
> ![IE11](https://user-images.githubusercontent.com/52488258/67024630-a3150780-f0fc-11e9-9d3d-d40123b2ca6b.PNG)
> 
> Just tested on one of the Windows machines, and it looks good to me. Even shows the Flash warning as expected.

@gavinspence  Great! This test is passed and I'll move it on :) Thanks for your help.
